### PR TITLE
EICNET-1546: Fix group block/publish operation links

### PR DESCRIPTION
**Pull request by 4hardfire**
### Fixes

- Fix block/publish operation links so that the destination URL takes into account the base path.

### Tests

- [ ] Go to a published group page
- [ ] Click on block
- [ ] Add a reason a press block
- [ ] You should be redirected back to the group page and you shouldn't get any error like: `Redirects to external URLs are not allowed by default, use \Drupal\Core\Routing\TrustedRedirectResponse`

### CODE REVIEW

File: lib/modules/eic_flags/src/Hooks/EntityOperations.php [hunk 1]
```diff
@@ -165,6 +165,16 @@ public function getOperations(EntityInterface $entity) {
       ];
     }
 
+    if ($entity->hasLinkTemplate('block-entity')) {
```
Comment (Dromadoros) [inline lib/modules/eic_flags/src/Hooks/EntityOperations.php:168]:
```suggestion
    if ($entity->hasLinkTemplate('block-entity') && $entity->toUrl('block-entity')->access()) {
```

File: lib/modules/eic_flags/src/Hooks/EntityOperations.php [hunk 2]
```diff
@@ -184,6 +194,17 @@ public function getAdminOperations(EntityInterface $entity) {
       return $handler->getActions($entity);
     }
 
+    $operations = [];
+    if ($entity->hasLinkTemplate('block-entity')) {
```
Comment (Dromadoros) [inline lib/modules/eic_flags/src/Hooks/EntityOperations.php:198]: same, just add &&

Comment (4hardfire) [reply lib/modules/eic_flags/src/Hooks/EntityOperations.php]: Done!

File: lib/modules/eic_flags/src/Hooks/EntityOperations.php (hunk 1 above)
Comment (4hardfire) [reply lib/modules/eic_flags/src/Hooks/EntityOperations.php]: Done!

**Review by Dromadoros — APPROVED** [review]

**Pull request merged**